inc/block-patterns.php

```diff
@@ -88,12 +87,12 @@ function twentytwentytwo_register_block_patterns() {
 		'hidden-404',
 		'hidden-bird',
 		'hidden-heading-and-bird',
-		'page-about-big-image-and-buttons',
 		'page-about-media-left',
 		'page-about-simple-dark',
 		'page-about-media-right',
 		'page-about-links',
 		'page-about-links-dark',
+		'page-about-big-image-and-buttons',
```

Comment:
Should the file be renamed to `page-about-large-image-and-buttons` for consistency?

Reply:
Good call! I'll make that change and merge. 👍